graphql/src/test/groovy/io/jmix/graphql/AbstractGraphQLTest.groovy

```diff
@@ -49,14 +53,40 @@ import test_support.TestContextInitializer
                 GraphQLTestAutoConfiguration.class,
                 JacksonAutoConfiguration.class
         ]
-)@TestPropertySource("classpath:/test_support/test-app.properties")
+)
+@TestPropertySource("classpath:/test_support/test-app.properties")
 class AbstractGraphQLTest extends Specification {
 
+    @Autowired
+    private DataManager dataManager
+
     protected TransactionTemplate transaction
 
     @Autowired
     protected void setTransactionManager(PlatformTransactionManager transactionManager) {
         transaction = new TransactionTemplate(transactionManager)
         transaction.setPropagationBehavior(TransactionDefinition.PROPAGATION_REQUIRES_NEW)
     }
+
+    def "data is initialized"() {
```

Comment:
I think we should not write test methods in this class. Such methods could run several times in case of inheritance (one time for each successor). 
Need to be moved in separate test class.